contrib/create-waku/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "outDir": "dist",
+    "rootDir": "./",
+    "strict": true,
+  },
+  "include": ["./"],
+  "exclude": ["node_modules", "dist"]
+}
```

Comment:
Let's make it consistent with the project config. we use module=nodenext for example.
Or, can we simply remove this file?

Reply:
Yeah, removing this is fine for now.